cms/migrations/0026_certificate_index_page_slug.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.15 on 2022-08-23 12:37
+
+import logging
+
+from django.db import migrations
+from wagtail.core.models import Page
+from django.contrib.contenttypes.models import ContentType
+
+log = logging.getLogger(__name__)
+
+
+def fix_certificate_index_page(apps, schema_editor):
+    """
+    Get HomePage and Create or update Certificate Index Page. Fix slug name if already created
+    """
+    from cms import models as cms_models
+    from cms.constants import CERTIFICATE_INDEX_SLUG
+
+    home_page = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(cms_models.HomePage), live=True
+    ).first()
+    if home_page:
+        try:
+            home_page.get_specific()
+            certificate_index = cms_models.CertificateIndexPage.objects.first()
+
+            if certificate_index and certificate_index.slug != CERTIFICATE_INDEX_SLUG:
+                certificate_index.slug = CERTIFICATE_INDEX_SLUG
+                certificate_index.save()
+
+            if not certificate_index:
+                cert_index_content_type, _ = ContentType.objects.get_or_create(
+                    app_label="cms", model="certificateindexpage"
+                )
+                certificate_index = cms_models.CertificateIndexPage(
+                    title="Certificate Index Page",
+                    content_type_id=cert_index_content_type.id,
+                    slug=CERTIFICATE_INDEX_SLUG,
+                )
+                home_page.add_child(instance=certificate_index)
```

Comment:
I think we already have a api/utility function to [ensure certificates index pages](https://github.com/mitodl/mitxonline/blob/main/cms/api.py#L192), I'd suggest using that instead of re-writing the whole logic here. Or if there is any specific reason for this, let me know.

Reply:
fixed